tools/ubuntu-setup/all.sh

```diff
@@ -10,15 +18,35 @@ echo "*** installing basics"
 echo "*** installing python dependences"
 "$SCRIPTDIR/pip.sh"
 
-echo "*** installing java"
-"$SCRIPTDIR/java8.sh"
-
-echo "*** install scala"
-"$SCRIPTDIR/scala.sh"
+u_release="$(lsb_release -rs)"
 
-echo "*** installing docker"
-"$SCRIPTDIR/docker.sh"
+echo "*** installing java"
```

Comment:
No need for this, let's install openjdk for any linux ubuntu 14 or 16

Reply:
Unfortunately, OpenJDK 8 is not available in trusty without jumping through the same hoops as 'java.sh' already does.

Reply:
Could you modify java.sh to pump and install OpenJDK ?